tests/tools/internal-rules/no-invalid-meta.js

```diff
@@ -241,6 +241,40 @@ ruleTester.run("no-invalid-meta", rule, {
                 line: 2,
                 column: 5
             }]
+        },
+
+        /*
+         * Rule doesn't export anything: Should warn on the Program node.
+         * See https://github.com/eslint/eslint/issues/9534
+         */
+
+        /*
+         * Should be invalid, but will currently show as valid due to #9534.
+         * FIXME: Uncomment when #9534 is fixed in major release.
```

Comment:
Can we add a note about this to the [5.0.0 milestone](https://github.com/eslint/eslint/projects/4) so we don't forget?

Reply:
:+1:, how would I go about doing that?

Reply:
Looks like we should make an issue to list these kinds of changes or just make an issue for the change itself.

Reply:
If you look at a column on the milestone page, and then click on the "+" near the top-right of the column, it allows you to add a note with arbitrary text.

Reply:
Thanks, I've added a note. I can convert into an issue if needed.